src/client/render.tsx

```diff
@@ -9,6 +9,7 @@ import type { RendererContext } from 'vscode-notebook-renderer';
 import { concatMultilineString } from './helpers';
 import { getTransform } from './transforms';
 import { OpenImageInPlotViewer, SaveImageAs, IsJupyterExtensionInstalled } from './constants';
+import { noop } from 'underscore';
```

Comment:
I'd suggest creating `const noop = () => {}` in the constants file, thats better than pulling in `underscore`, unless we can just import what we need using  something like `from 'underscore/noop';`

Reply:
whoops, didn't check what I imported there. Good call